packages/sdk/src/messages/NomadMessage.ts

```diff
@@ -257,12 +269,22 @@ export class NomadMessage<T extends NomadContext> {
    *
    * @returns The timestamp at which a message can confirm
    */
-  async confirmAt(): Promise<number | undefined> {
-    if (!this.eventCache.confirmAt || this.eventCache.confirmAt === 0) {
+  async confirmAt(): Promise<Date | undefined> {
+    if (!this._confirmAt || this._confirmAt.valueOf() === 0) {
       await this._events();
+      if (this.eventCache.relay) {
+        const intTs = parseInt(this.eventCache.relay.timestamp); // may throw
+        if (intTs <= 946684800000) {
```

Comment:
What is this timestamp number?

Reply:
temporary, but it is check if it is 2000-01-01T00:00:00.000Z though, it is just not much relevant